Test Cases/Licensure Unit License Application/Appeal Denied License Application - Invalid denial code.tc

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<TestCaseEntity>
+   <description>On this Test Case the Admin User approve the License Application.</description>
+   <name>Appeal Denied License Application - Invalid denial code</name>
```

Comment:
The name of the Test Case should be "Submit appeal with invalid denial code from Public link"

Reply:
Done